markdown-mode.el

```diff
@@ -4161,12 +4161,19 @@ these cases, indent to the default position.
 Positions are calculated by `markdown-calc-indents'."
   (interactive)
   (let ((positions (markdown-calc-indents))
+        (cursor-pos (current-column))
+        (_ (back-to-indentation))
```

Comment:
This looks a little gross, but the alternative is another layer of let-bindings; let me know if you see a third path, or prefer something else!